src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpSinglelineCheck.java

```diff
@@ -71,17 +71,46 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
- * To configure the check to find trailing whitespace at the end of a line,
- * with some <i>slack</i> of allowing two occurrences per file:
+ * To configure the check to find occurrences of 'System.exit('
+ * with some <i>slack</i> of allowing only one occurrence per file:
  * </p>
  * <pre>
  * &lt;module name="RegexpSingleline"&gt;
- *   &lt;property name="format" value="\s+$"/&gt;
+ *   &lt;property name="format" value="System.exit\("/&gt;
  *   &lt;!-- next line not required as 0 is the default --&gt;
  *   &lt;property name="minimum" value="0"/&gt;
- *   &lt;property name="maximum" value="2"/&gt;
+ *   &lt;property name="maximum" value="1"/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class MyClass {
+ *      void myFunction() {
+ *          try{
+ *             doSomething();
+ *          } catch (Exception e) {
+ *             System.exit(1) // OK, as only there is only one occurrence.
+ *          } finally {
+ *              System.println("success!")
+ *          }
+ *      }
+ *      void doSomething(){};
+ * }
+ * </pre>
+ * <pre>
+ * class MyClass {
+ *     void myFunction() {
+ *         try{
+ *             doSomething();
+ *         } catch (Exception e) {
+ *             System.exit(1)
+ *         } finally {
+ *             System.exit(0) // violation, as there are more than one occurrence.
```

Comment:
please move ` System.exit(0)` to be after `doSomething`.

Reply:
> please move ` System.exit(0)` to be after `doSomething`.

done